constants/regex.ts

```diff
@@ -1,2 +1,4 @@
 export const TRUNCATE_ETH_ADRESS_REGEX = /^(0x[a-zA-Z0-9]{4})[a-zA-Z0-9]+([a-zA-Z0-9]{4})$/;
 export const TRUNCATE_NFT_ADRESS_REGEX = /^(0x[a-zA-Z0-9]{4})[a-zA-Z0-9]+_[a-zA-Z0-9]+_([0-9]+)$/;
+
+export const SAFE_CONTENT_RELATIVE_PATH_REGEX = /^[a-zA-Z0-9_-]+\/[a-zA-Z0-9_-]+\.md$/;
```

Comment:
I believe this will include only first directory level. This should accept any amount of levels and exclude ".." path